drivers/storage/portworx/component/autopilot.go

```diff
@@ -189,6 +190,12 @@ func (c *autopilot) createClusterRole() error {
 					Resources: []string{"*"},
 					Verbs:     []string{"*"},
 				},
+				{
```

Comment:
Does autopilot really need a PSP?

Reply:
Yes, if PodSecurityPolicy admission plugin is enabled each controller have to use any PSP to create pods.
https://kubernetes.io/docs/concepts/policy/pod-security-policy/#authorizing-policies